packages/cli/src/cmds/beacon/handler.ts

```diff
@@ -182,6 +182,11 @@ export async function beaconHandlerInit(args: BeaconArgs & GlobalArgs) {
   // Add detailed version string for API node/version endpoint
   beaconNodeOptions.set({api: {commit, version}});
 
+  // Set noValidatorCustody to true if supernode is set
```

Comment:
a more informative comment would be helpful here, ie. one that explains why we need to disable validator custody if we are a supernode

Reply:
Updated to ```Disable dynamic custody updates for supernodes since they must maintain custody of all custody groups regardless of validator effective balances```